lib/ansible/modules/storage/purestorage/purefa_volume.py

```diff
@@ -151,31 +198,41 @@ def main():
     argument_spec.update(
         dict(
             name=dict(required=True),
+            target=dict(),
+            overwrite=dict(default='false', type='bool'),
```

Comment:
Ansible imho does the right thing (I guess) but since this is `type=bool`, you can use python bool here, like `default=False` (without quotes).